### ISSUE TITLE
PEP 541 Request: qrcode

### ISSUE THREAD

**Issue by septatrix** [pypa/pypi-support#807]
**Project to be claimed**

`qrcode`: https://pypi.org/project/qrcode/

**Your PyPI username**

`Septatrix`: https://pypi.org/user/Septatrix/

**Reasons for the request**

The project seems abandoned. The latest release was at the start of this year and in the last 4 years there were only two other releases. On Github there are no recent comments from the maintainers on issues or pull requests. It is not marked as ready for Python 3.7-3.10 and no progress towards this is made (even though pull requests exist).

**Maintenance or replacement?**

I intend to maintain the current code. This will be done by forking the github repository and merging pull requests. After eliminating code dept I plan to add new features requested in the github issues.

**Contact and additional research**

The main contributor and developer appears to be [SmileyChris](https://pypi.org/user/SmileyChris/).
He did not respond to a ping on github and I got no response from writing a mail to _lincolnloop_.

**Comment by septatrix** [comment]
Any update on this?

**Comment by yeraydiazdiaz** [comment]
Hello, PEP 541 requires moderators to attempt contact with the owners for 6 weeks before proceeding with the transfer. I will contact user @SmileyChris here and also by email, hopefully we will hear back soon.

**Comment by SmileyChris** [comment]
Hello! I have indeed been rather slack on keeping up with the qrcode development, and extra maintainers sounds good.

I didn't note any ping on github (but I get drowned in notifications) and I didn't notice any mail to lincolnloop.
@septatrix, ping me on my private email (smileychris@gmail) and we'll connect about how to move forwards. At the minimum, I'm fine with adding you as a maintainer to both pypi and the repo. We could also look at creating a new github organization and move the repo there potentially.

In summary: I did notice this and am happy to share maintenance to avoid the need to fork.

**Comment by yeraydiazdiaz** [comment]
Thanks for replying @SmileyChris, I will close this as contact has been made.

**Comment by septatrix** [comment]
Just as a general information: Shortly after this development started to get going again so I did not initiate contact further. If in the future development starts to slow down again or new maintainers are needed I will reach out to the mail address kindly left my chris
